### DISCUSSION TITLE
【灌水】如何提升绳网等级

### DISCUSSION THREAD

**Discussion by SnowBall12345678** [share121/inter-knot#2875] [其他]
![Image_1695098403275](https://github.com/user-attachments/assets/c9be7270-277e-4468-b854-26980372b487)

如何提高绳网等级，我要成为绳网萝卜高手！
（附图是我老婆）

**Comment by panda114514667** [comment]
帖子不错，我蹭了

**Comment by Huanshenxuanlong** [comment]
多接任务多做委托，现在法厄同号被封了正是新人崛起的时候

**Comment by SnowBall12345678** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by SnowBall12345678** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/7981625358385b482d2c84d02de195cf.png" alt="观星身体" width="75" >
<img src="https://img-static.mihoyo.com/communityweb/upload/96fcb4abaed65bddc3ab84f012b80c25.png" alt="观星长腿" width="75" >
